engine/src/main/java/io/camunda/zeebe/engine/processing/processinstance/ProcessInstanceMigrationMigrateProcessor.java

```diff
@@ -114,6 +114,23 @@ public void processRecord(final TypedRecord<ProcessInstanceMigrationRecord> comm
       return;
     }
 
+    final DeployedProcess sourceProcessDefinition =
```

Comment:
:upside_down_face: Should we also rename [the variable on line 106](https://github.com/camunda/zeebe/pull/15519/files#diff-7388b163a2f978814a720c3ee81b65fd415dac06abf2aefa99881eacaab0b787R106) `targetProcessDefinition` to make the code more readable?